### PR TITLE
feat: new daily ci enroll form (#29)

### LINKED ISSUE #29: Recreate daily check-in enrollment
Ponder on the possibilities of leaving the initial offer in or out. I'm not yet fixed on the best idea for this.

### PR DESCRIPTION
**Pull request by NetBKarine**
## Description

Got the "do you want to enroll" question out of the form. For this and the "digressions" when collecting the phone number, I had to isolate the daily check-in enrollment rules, and did it with an action that does nothing, to be used as a marker to come back to the correct form. It works well, like a checkpoint but with more flexibility.

Other things that look like digressions I had to leave in the form even though the plan was to get them out, because it is cleaner this way when the "digression" affects how we set a slot or how we ask for it. We can't ask the question inside a digression rule and automatically come back to the form using the user's answer, as I had thought would be possible.

Also, added a `make clean` task. The description was there, but not the task itself.

## Related issues

closes #29

## Checklist

- [X] PR title follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) guidelines 
- [X] All relevant PR sections are populated, irrelevant ones are removed

### CODE REVIEW

File: core/config/config.en.yml [hunk 1]
```diff
@@ -3,9 +3,10 @@
 language: en
 
 pipeline:
-  - name: ConveRTTokenizer
-  - name: ConveRTFeaturizer
-    case_sensitive: false
+  - name: WhitespaceTokenizer
+  # - name: ConveRTTokenizer
+  # - name: ConveRTFeaturizer
+  #   case_sensitive: false
```
Comment (NetBKarine) [inline core/config/config.en.yml:9]: Y'a un issue avec le modèle pour le ConveRT, je vais ajouter un commentaire avec l'issue, j'ai oublié. En attendant que ce soit à jour dans rasa 2 ET qu'on update, on va laisser ça de même.

**Review by mbrisebo-perso — CHANGES_REQUESTED** [review]
À part un commentaire sur un bout de code suspect, LG(reat)TM

File: action-server/covidflow/actions/daily_ci_enroll_form.py [hunk 2]
```diff
@@ -280,20 +276,23 @@ async def _validate_validation_code(
...
+    (reached_max_errors, error_events) = _check_error_counter(
+        VALIDATION_CODE_SLOT, CODE_TRY_COUNTER_SLOT, tracker, dispatcher
+    )
+
+    if value == "did_not_get_code":
+        return (
+            error_events
+            if reached_max_errors
+            else error_events + [SlotSet(NO_CODE_SOLUTION_SLOT, None)]
+        )
```
Comment (mbrisebo-perso) [inline action-server/covidflow/actions/daily_ci_enroll_form.py:293]: OK, je comprends mieux...
C'est une syntaxe propre à Python que je n'avais pas encore rencontrée.

**Review by mbrisebo-perso — APPROVED** [review]

File: action-server/covidflow/actions/daily_ci_enroll_form.py (hunk 2 above)
Comment (NetBKarine) [reply action-server/covidflow/actions/daily_ci_enroll_form.py:293]: Oui, c'est le "shorthand if" de python. Assez utile, un peu bizarre, comme à peu près tout en python.

**Pull request merged**